### ISSUE TITLE
[Hint] Support multi query SQL hint.

### ISSUE THREAD

**Issue by dongzl** [arana-db/arana#354]
**What would you like to be added**:

**Why is this needed**:

**Comment by jjeffcaii** [comment]
Partial dependency feature: https://github.com/arana-db/arana/pull/402

**Comment by dongzl** [comment]
ref #585

**Comment by maronghe** [comment]
Yes, @marioooo0 and I  discussed HINT this morning and found that only a few HINT capabilities have been implemented. 

And we need more HINT-related content later, but for now it may not be the focus of the current version 0.2.0 I think.

And we have a problem. 

For arana's HINT, the sharding method is specified, but for mysql's own HINT, it seems that it has not been parsed, or `PARSER` has not yet supported it? Not sure, so leave a message below the issue.
